tests/regression/roles/sonic_bgp_af/defaults/main.yml

```diff
@@ -298,7 +288,3 @@ tests:
         vrf_name: default
         address_family:
           afis:
-  - name: test_case_07
-    description: Delete3 BGP AF properties
-    state: deleted
-    input: []
```

Comment:
Was this test case failing? (It is common to do a "delete all" at the end of a test. Usually, it doesn't cause a failure.)

Reply:
Yes it was failing the after was already undefined

Reply:
We need to do some analysis to figure out the reason for the failure.

Reply:
Test case 7 was failing due to being undefined since the previous test case already deleted all existing config. I removed config from test case 5 so that the delete all would have something to delete.